contracts/extensions/ede000-governance-token.clar

```diff
@@ -36,6 +38,7 @@
 (define-public (edg-transfer (amount uint) (sender principal) (recipient principal))
 	(begin
 		(try! (is-dao-or-extension))
+		(asserts! (not (var-get transfer-lock)) err-transfer-is-locked)
```

Comment:
Hold up, I just noticed this. The `edg-` prefixed functions are meant to be DAO privileged functions, meaning the DAO should always be able to act on the token in any way it sees fit. I therefore do not think this guard is appropriate. (It is pointless regardless, because a proposal can call `(set-transfer-lock false)` before calling into `edg-transfer`.)